src/components/Blocks/Banner/components/BannerDescriptionTypography/BannerDescriptionTypography.tsx

```diff
@@ -0,0 +1,18 @@
+import { usePlatform } from 'hooks/usePlatform';
+
+import { Caption } from 'components/Typography/Caption/Caption';
+import { Subheadline } from 'components/Typography/Subheadline/Subheadline';
+import { TypographyProps } from 'components/Typography/Typography';
+
+export interface BannerDescriptionTypographyProps extends Omit<TypographyProps, 'level'> {
+}
+
+export const BannerDescriptionTypography = (props: BannerDescriptionTypographyProps) => {
+  const platform = usePlatform();
+
+  if (platform === 'base') {
```

Comment:
Great, thank you for this changes. The last thing, it should be ios here, because base is fallback